index.html

```diff
@@ -35,6 +35,170 @@ <h2 class="green">I’m a software developer! </h2></div>
         <li><a href=""><img src="icons/angellist.svg"></a></li>
       </ul>
     </div>
-      </section>
+    <img src="icons/Disabled.svg" alt="next">
+    <div class="works">
```

Comment:
- [ ] As suggested by the previous reviewer, consider replacing this div at line 39 with `section` tag as it is semantically corrected and helps in SEO as well.